From e49234e0a3c17945f8db0c3e68f2697b3fd6af68 Mon Sep 17 00:00:00 2001
From: reshu <reshu@gmail.com>
Date: Fri, 4 Oct 2019 11:18:47 +0530
Subject: [PATCH 2/2] patched

---
 patches/patchfile | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 patches/patchfile

diff --git a/patches/patchfile b/patches/patchfile
new file mode 100644
index 0000000..190b01e
--- /dev/null
+++ b/patches/patchfile
@@ -0,0 +1,2 @@
+hii
+this is patch..
-- 
2.17.1

